wadas/domain/whatsapp_notifier.py

```diff
@@ -87,7 +94,11 @@ def send_whatsapp_message(self, img_path):
                 if message_response.status_code == 200:
                     logger.info("WhatsApp notification sent!")
                 else:
-                    logger.error(message_response.status_code, message_response.text)
+                    err_message = (
+                        f"Failed to send WhatsApp notification: "
```

Comment:
Redundant f-string (nothing is interpolated inside the string).

Reply:
it's a multiline f string (due to line too long fix):

                    err_message = (
                        f"Failed to send WhatsApp notification: "
                        f"{message_response.status_code}, {message_response.text}"
                    )

Reply:
Yes, but actually only the second one needs to interpolate the string and a value: the first one is just a plain string, and it doesn't use the interpolation abilities of f-strings.